woocommerce/woocommerce-android
# Analytics hub product name can exceed the area

Closes: #7812
Closes: #8030


### Description

* Adds placeholder for the products on the Analytics screen
* Improves layout so long text won't break that 

### Testing instructions

* Open analytics screen with different products on it
* Notice that it looks correct now

### Images/gif

#### Before:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/4923871/210230278-c723c19c-720d-41eb-976e-e844eb6b6409.png">


#### After:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/4923871/210230201-55902c40-a27f-4a93-8a04-b35543b10718.png">
